appmanifest/shortcuts-member/shortcuts-member-skip-for-empty-name.webmanifest

```
@@ -0,0 +1,17 @@
+{
+    "shortcuts": [{
```

We might need to add some additional things to make sure Windows can install this app. 

@aarongustafson can probably advice on what's needed. 

```suggestion
    "name": "Shortcut test",
    "scope": "/", 
    "shortcuts": [{
```